deploy/olm-catalog/keycloak-operator/15.1.0/manifests/keycloakclients.keycloak.org.crd.yaml

```diff
@@ -0,0 +1,843 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: keycloakclients.keycloak.org
+spec:
+  group: keycloak.org
+  names:
+    kind: KeycloakClient
```

Comment:
This needs to be identical with [`keycloak.org_keycloakclients_crd.yaml`](https://github.com/keycloak/keycloak-operator/blob/2deeb25d373c7dbdb67d702244620269502fcacb/deploy/crds/keycloak.org_keycloakclients_crd.yaml).

Reply:
yes, which is the problem here ?

Reply:
It's not identical. :)
![Screenshot 2021-12-14 at 15 03 30](https://user-images.githubusercontent.com/13906855/146013186-377aced5-4389-47f9-bc43-3997dfda5170.png)